cmd/server-manager/views/pages/live-timing.html

```diff
@@ -232,6 +232,24 @@ <h4>Stored Times</h4>
             </div>
         </div>
 
+        <div class="row mt-2">
+            <div class="col-lg-7 col-md-12"></div>
+            <div class="col-lg-5 col-md-12">
+                <div class="card card-body" id="chat-container">
```

Comment:
could all of this be in the div that is the parent of the map? if the live timings list gets really long, the chat will be below all of that, with a big gap between it and the map, e.g.

![image](https://user-images.githubusercontent.com/306873/86033561-064b8d80-ba31-11ea-9e44-d510a2536de2.png)

(image totally not necessary, i just had fun making it)